datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/config.py

```diff
@@ -51,6 +52,8 @@ def config(ctx, check, sync, verbose):
     else:
         checks = process_checks_option(check, source='valid_checks', extend_changed=True)
 
+    is_core_check = ctx.obj['repo_choice'] == 'core'
```

Comment:
Small nit that can be for another PR. Would it be worth considering checking not only whether the repository configuration in ddev is named 'core,' but also examining whether the repository path contains 'integrations-core' in its folder path? Example like:

`"integrations-core" in ctx.obj['repos'][ctx.obj['repo_choice']]`

I tend to name things pretty randomly and sometimes I switch to forks. Not a big blocker either way. I can just go back to naming my repo core.

Reply:
I'm adding a note to take that into account when we'll migrate the command to ddev (it's still in datadog_checks_dev). Would that work for you?

Reply:
Sounds good!